src/vitepress/config/baseConfig.js

```diff
@@ -56,10 +56,6 @@ module.exports = async () => ({
     ]
   ],
 
-  markdown: {
-    highlight: await require('./highlight')()
-  },
```

Comment:
May I know why the lighlight config was removed? The demo looks like this:

![image](https://user-images.githubusercontent.com/206848/187018751-9b25c541-c94a-4c05-a9ae-905717c3d37a.png)

![image](https://user-images.githubusercontent.com/206848/187018758-bd4a3ff8-8fef-47ed-bded-26f4ca4d5421.png)

http://localhost:3000/guide/introduction.html

Reply:
>  The demo looks like this:

Are you sure you are on this PR branch (and VitePress alpha is installed)? This is the output I'm getting (https://deploy-preview-70--vue-vitepress-theme.netlify.app/guide/introduction.html):

![image](https://user-images.githubusercontent.com/40380293/187019232-11f6d8a8-eb81-4d0f-a764-cd1c384999f9.png)

That highlighter was added because in v0.x VitePress was using highlight.js. In v1.x VitePress is also using shiki, so there is no need to duplicate the logic.

Reply:
Oh, I see. No problem here. Thanks for pointing this out.